deps/rabbitmq_shovel/test/dynamic_SUITE.erl

```diff
@@ -262,6 +263,36 @@ exchange(Config) ->
                              <<"queue">>, <<"hello">>)
       end).
 
+
+missing_dest_exchange(Config) ->
+    with_ch(Config,
+        fun (Ch) ->
+            amqp_channel:call(
+            Ch, #'queue.declare'{queue = <<"src">>, % E: record 'queue.declare' undefined
```

Comment:
I'm not sure what these comments are meant to communicate?

Reply:
Thank you and sorry about that, it looks like remains of a poorly configured linter on my end. Should've spotted that!